Code/UnitTests/RendererTest/Advanced/AdvancedFeatures.cpp

```diff
@@ -154,6 +160,23 @@ ezResult ezRendererTestAdvancedFeatures::InitializeSubTest(ezInt32 iIdentifier)
     m_pProtocol->Send(&msg);
   }
 
+  if (iIdentifier == ST_Tessellation)
+  {
+    {
+      ezGeometry geom;
+      geom.AddSphere(0.5f, 3, 2);
+
+      ezMeshBufferResourceDescriptor desc;
+      desc.AddStream(ezGALVertexAttributeSemantic::Position, ezGALResourceFormat::XYZFloat);
```

Comment:
I think the two addstreams are redundant, because it's already set up through the geometry

Reply:
Doesn't work, `ezMeshBufferResourceDescriptor::AllocateStreams` will assert that I need to call `AddStream` first. I think the idea is that `AllocateStreamsFromGeometry` only fills the streams that you have actually previously requested?

Reply:
Okay, possible. I haven't looked at the implementation,I thought it did everything for you.